### ISSUE TITLE
Documentation Please

### ISSUE THREAD

**Issue by seanmtracey** [ftlabs/Screens-Chrome-App#5]
How do we build this app?

**Comment by JakeChampion** [comment]
I think you made this whole app

**Comment by JakeChampion** [comment]
I believe it is `npm run build` for a local copy and `npm run deploy` for the dist version

**Comment by seanmtracey** [comment]
Yeah, the issue is a reminder from me to me to get me to write me documentation for the app. And yes, those are the commands that should do the trick...
